exercism/wordy/wordy.src.js

```diff
@@ -0,0 +1,38 @@
+class WordProblem {
+  constructor (question){
+    this.question = question.replace(/ by/g, '').replace('What is ', '').replace('?', '');
+    this.operators = {
+      "plus": function (a,b) {
+        return a + b;
+        },
+      "minus": function (a,b) {
+        return a - b;
+        },
+      "multiplied": function (a,b) {
+        return a * b;
+        },
+      "divided": function (a,b) {
+        return a / b;
+        }
+      };
+  };
+
+  answer() {
+    var splitQuestion = this.question.split(' ');  //split question into an array
+    var newOperator = splitQuestion[1];            //extract first operator
+    var secondOperator = splitQuestion[3];         //extract second operator
+    if (isNaN(splitQuestion[2])) {
+      throw new ArgumentError();
+      } else {
+      var answer1 = this.operators[newOperator](parseInt(splitQuestion[0]),parseInt(splitQuestion[2]));
```

Comment:
It is very hard to read what is going on here. Looks like you are extracting the operands, and applying the first operator. 

An alternative way to do this would be to set the `operand1`, `operand2`, `operator1` & `operator2` as attributes of your class. This could be set in your constructor. And then your function `answer` just uses these attributes to compute the result.